app/initializers/ember-data-twitch.js

```diff
@@ -1 +1,24 @@
-export { default } from 'ember-data-twitch/initializers/ember-data-twitch';
+import TwitchAdapter from 'ember-data-twitch/adapters/twitch';
+import TwitchSerializer from 'ember-data-twitch/serializers/twitch';
+import ENV from '../config/environment';
+import TwitchConfig from 'ember-data-twitch/configuration';
+
+export function initialize(/* application */) {
+
+  // Support initializer signature from before 2.1.0
+  // See http://emberjs.com/deprecations/v2.x/#toc_initializer-arity
+  const application = arguments[1] || arguments[0];
+  const twitchConfig = ENV['ember-data-twitch'];
```

Comment:
Prefer `let` over `const`. Here's a good explanation from mixonic: https://madhatted.com/2016/1/25/let-it-be